infra/readme.md

```diff
@@ -0,0 +1,30 @@
+# Deploying the infrastructure
+
+You can deploy the Kernel Memory infrastructure to Azure by clicking the button below. This will create required resources. We recommend to create a new resource group for this deployment.
+
+Resource that deployed have an opinionated set of configurations. You can modify services on Azure portal or you can reuse Bicep file and provide your customisations `infra/main.bicep`.
```

Comment:
The comment suggests to edit the bicep file, but the link points to the json file. Is it possible to change the link and point it to the bicep file? Otherwise we should include instructions about how to make changes and rebuild the JSON file.